src/screens/SignIn.js

```diff
@@ -46,38 +44,57 @@ export default class SignIn extends Component<Props, State> {
     });
   };
 
-  onLogin = async () => {
-    const { email } = this.state;
-
+  googleLogin = async () => {
     try {
       await this.asyncState({ isSending: true });
+      await GoogleSignin.configure();
+      const data = await GoogleSignin.signIn();
 
-      await firebase.auth().sendSignInLinkToEmail(email, {
-        url: "https://jupiter-9b937.firebaseapp.com",
-        iOS: {
-          bundleId: "com.taxischoferes"
-        },
-        android: {
-          packageName: "com.taxischoferes",
-          installApp: true,
-          minimumVersion: "1"
-        },
-        handleCodeInApp: true
-      });
+      const credential = firebase.auth.GoogleAuthProvider.credential(data.idToken, data.accessToken)
 
-      await AsyncStorage.setItem("emailForSignIn", email);
+      await firebase.auth().signInWithCredential(credential);
```

Comment:
Esta línea no muestra algún warning? Es que recuerdo que si me hizo eso y en la documentación muestra esto: `This method is deprecated.`.

Reply:
Parece que se debe reemplazar con `signInAndRetrieveDataWithCredential`.

Reply:
no, quise ocupar el método signInAndRetrieveDataWithCredential, y ese si me mostraba ese warning y lo cambie por ese que tiene

Reply:
Me parece que te regresa la información del usuario. Pudes confirmar con esto:
```
const user = await firebase.auth().signInWithCredential(credential);
console.warn(user)
```